src/utils.js

```diff
@@ -74,15 +74,20 @@ function createCSVDownload(columns, data, options) {
     ? options.onDownload(buildHead, buildBody, columns, data)
     : `${CSVHead}${CSVBody}`.trim();
 
-  if (options.onDownload && csv === false) {
-    return;
+  const downloadHasBeenAborted = csv === false;
+  if (downloadHasBeenAborted) {
+    return null;
   }
 
+  return csv;
+}
+
+function downloadCSV(csv, filename) {
```

Comment:
Great idea to break this out into its own function! Helps a lot with the testing.